modules/contact-form/tracks-events.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+use Automattic\Jetpack\Tracking;
+
+function tracks_record_grunion_pre_message_sent( $post_id, $all_values, $extra_values ) {
```

Comment:
Should this function use a `jetpack_` prefix?

Reply:
👍